docs/_articles/en/codebuilder/about.md

```diff
@@ -2,27 +2,21 @@
 title: Code Builder Overview
 lang: en
 ---
+
 ## Overview
-Salesforce Code Builder is a web-based integrated development environment that has all the power and flexibility of Visual Studio Code, Salesforce Extensions for VS Code, and Salesforce CLI in your web browser. Code Builder provides a modern developer experience for all developers, regardless of expertise level. Code Builder makes it easy for admins and developers alike to work in the cloud without having to worry about downloading software, setup or your machine specs.
 
-Code Builder is development environment that can be spun up in seconds, and is custom to your Salesforce org and your specified project. Through Code Builder, access your favorite Salesforce languages and frameworks, such as Apex, SOQL, Visualforce, Aura, and Lightning Web Components. Access rich developer tools such as debuggers for Apex, Linting and so on.
```

Comment:
@sbudhirajadoc - I suggest changing the first sentence to "....spun up in minutes, and is your personal environment."

I'd prefer to set the expectation on loading min minutes because that's current reality.  Since they can change orgs or projects within Code Builder, I'm worried that the current wording of "custom to your Salesforce org and specified project" might be misleading.  (sorry i missed this on my first review)

Reply:
Wait, you're not seeing an updated version of the doc. The [doc preview](https://deploy-preview-5013--salesforcedx-vscode.netlify.app/tools/vscode/en/codebuilder/about) shows the updated version where is says "quickly" instead of seconds.